_posts/0000-01-02-ca99uk.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
You could have some information about what is on the second slide

  + Agenda
  + Scope